packages/labs/react/src/create-component.ts

```diff
@@ -215,17 +215,15 @@ export const createComponent = <I extends HTMLElement, E extends Events = {}>(
           this._userRef = userRef;
         };
       }
-      // Filters class properties out and passes the remaining
-      // attributes to React. This allows attributes to use framework rules
-      // for setting attributes and render correctly under SSR.
+      // Save element props while iterating to avoid the need to iterate again
+      // when setting properties.
+      this._elementProps = {};
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       const props: any = {ref: this._ref};
-      // Note, save element props while iterating to avoid the need to
-      // iterate again when setting properties.
-      this._elementProps = {};
-      for (const [k, v] of Object.entries(this.props)) {
-        if (k === '__forwardedRef') continue;
```

Comment:
This might regress a previously solved issue
https://github.com/lit/lit/issues/2593

Reply:
It's pulled out in the destructuring above in a way that survives property renaming. There's also a test for that issue that's not failing.

Reply:
awesome!